libsplinter/src/admin/service/shared.rs

```diff
@@ -978,6 +978,24 @@ impl AdminServiceShared {
         )
     }
 
+    #[cfg(feature = "circuit-purge")]
+    fn purge_circuit(&mut self, circuit_id: &str) -> Result<(), ServiceError> {
```

Comment:
Yep, I wasn't sure when that data about the database/lmbd files should be communicated to the admin service. Would probably need to happen before the service is shutdown, right? So, would maybe need to hold that service information in the admin service so it can be used when the circuit is purged to actually clear out that data

Reply:
Added a story to implement what we had landed on to handle this this morning